app/controllers/home_controller.rb

```diff
@@ -9,5 +9,9 @@ def index
     @countries_collection.delete(["country_name", :COUNTRY_ISO_CODE])
     @countries_collection = @countries_collection.sort_by {|c| c[0]}
     @volunteer_crews = Volunteer.crews.collect{|k,v| [k.humanize, k]}
+
+    # Pre-selected New York
+    @states_collection = CS.states(:US).map { |c| [I18n.transliterate(c[1]), c[0]] }
+    @cities_collection = CS.cities(:NY, :US)
```

Comment:
Was this done elsewhere?

Reply:
This actually is in master already, but it seems like sometimes Github gets confused about the order of commits when it creates these comparisons - I've had it happen a few times

Reply:
The actual diff: https://github.com/panvol/pandemic-volunteers/compare/master...ewynx:remove-regex-phone-nr